docs/src/content/docs/commands/HGETALL.md

```diff
@@ -57,37 +66,36 @@ HGETALL user:1000
 ### Example 2: Retrieving from a non-existing key
 
 ```DiceDB
-HGETALL user:2000
+127.0.0.1:7379> HGETALL user:2000
 ```
 
 `Output:`
 
 ```
-(empty array)
+(empty list or set)
```

Comment:
This looks inconsistent to what I am getting on latest DiceDB and redis locally. Please confirm.

Reply:
Hey @apoorvyadav1111 - addressing all the comments mentioned above, I haven't taken the latest pull and the current state of my fork is of that when this PR was made, and here's the same scenario I executed today:

![image](https://github.com/user-attachments/assets/2071c483-c248-4002-8b7a-675e31aacba8)

Reply:
Took the latest pull, getting the same response as above

Reply:
I think the difference comes from using redis-cli or dice-cli. I would say keep it as empty array for now to keep it consistent across other docs.

Reply:
Cool @apoorvyadav1111 - keeping it empty array, lets keep it consistent as you mentioned. Will push the recommit soon.
Thanks